js/symbol/sprite_atlas.js

```diff
@@ -38,6 +38,8 @@ SpriteAtlas.prototype = {
 };
 
 SpriteAtlas.prototype.resize = function(newRatio) {
+    newRatio = newRatio > 1 ? 2 : 1;
```

Comment:
There's no standardization. Some browsers will change the ratio with the zoom level, some will not. The MS Surface Pro reports a default ratio of 0.87.